src/Storage/Storage.Management/Models/PSDataPolicy.cs

```diff
@@ -349,18 +390,15 @@ public PSDateAfterModification(DateAfterModification data)
             {
                 this.DaysAfterLastAccessTimeGreaterThan = Convert.ToInt32(data.DaysAfterLastAccessTimeGreaterThan);
             }
-            if (data.DaysAfterLastTierChangeGreaterThan is null)
-            {
-                this.DaysAfterLastTierChangeGreaterThan = null;
-            }
-            else
-            {
-                this.DaysAfterLastTierChangeGreaterThan = Convert.ToInt32(data.DaysAfterLastTierChangeGreaterThan);
-            }
         }
-        public DateAfterModification ParseDateAfterModification()
+        public Track2.Models.DateAfterModification ParseDateAfterModification()
         {
-            return new DateAfterModification(this.DaysAfterModificationGreaterThan, this.DaysAfterLastAccessTimeGreaterThan, this.DaysAfterLastTierChangeGreaterThan);
+            Track2.Models.DateAfterModification dateAfterModification = new Track2.Models.DateAfterModification();
+            dateAfterModification.DaysAfterLastAccessTimeGreaterThan = this.DaysAfterLastAccessTimeGreaterThan;
+            dateAfterModification.DaysAfterModificationGreaterThan = this.DaysAfterModificationGreaterThan;
+
```

Comment:
missed DaysAfterLastTierChangeGreaterThan

Reply:
Done. It's not supported yet, so added a TODO comment for it